addons/mail/static/src/drag_and_drop/dropzone_container.js

```diff
@@ -0,0 +1,62 @@
+/** @odoo-module **/
+
+import { Dropzone } from "@mail/drag_and_drop/dropzone/dropzone";
+
+import { Component, useExternalListener, useState, xml } from "@odoo/owl";
+
+export class DropzoneContainer extends Component {
+    static template = xml`
+        <t t-if="state.isDraggingFile">
+            <t t-foreach="[...props.dropzones]" t-as="dropzone" t-key="dropzone.id">
+                <Dropzone ref="dropzone.ref"/>
+            </t>
+        </t>
+    `;
+    static props = { dropzones: Object };
+    static components = { Dropzone };
+
+    isDraggingFile = false;
+
+    #dragCount = 0;
+
+    setup() {
+        this.state = useState({
+            isDraggingFile: false,
+        });
+        useExternalListener(document, "dragenter", this._onDragEnter);
+        useExternalListener(document, "dragleave", this._onDragLeave);
+        useExternalListener(document, "drop", this._onDrop);
+    }
```

Comment:
Methods are useless, can just use arrow functions immediately as param of `useExternalListener()`